common/supplemental/metaZones.xml

```diff
@@ -1892,6 +1892,167 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<mapZone other="Yerevan" territory="001" type="Asia/Yerevan"/>
 			<mapZone other="Yukon" territory="001" type="America/Whitehorse"/>
 		</mapTimezones>
+
+		<metazoneIds>
```

Comment:
Optional: I was thinking originally that this section could contain additional metadata about metazones, not just the ID mapping.  In that case, `<metazoneInfo>` or `<metazoneDetails>` might be an appropriate tag name.

Reply:
Thoughts on my optional comment?

```xml
<metazoneDetails>
    <metazoneDetail ... />
    <metazoneDetail ... />
</metazoneDetails>
```